From b2d86559ba2ec28d5ae11b777098b58c4ea38ba9 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Mon, 1 Apr 2019 19:45:21 +0900
Subject: [PATCH 791/909] Revert "arm64: dts: r8a77965: Connect SDHI to
 IPMMU-DS1"

commit 7d702c9baadf7c9578dab25ee5513f96fe7a6f5a from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This reverts commit 21258b113ddea10592c3c90979f269bd31bd0622.

Multiple IPMMUs can not be operated simultaneously due to H3 Ver.1.x,
H3 Ver.2.0, M3 Ver.1.x hardware restriction.

M3-N has no similar hardware restriction, but disables IPMMU for SoCs
that have hardware restriction.

As a result, This reverts commit 21258b113dde ("arm64: dts: r8a77965:
Connect SDHI to IPMMU-DS1") to keep IPMMU for SDHI{0..3} is disabled by
default.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77965.dtsi | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a77965.dtsi b/arch/arm64/boot/dts/renesas/r8a77965.dtsi
index e8381250df8b..8dd0dca6efcf 100644
--- a/arch/arm64/boot/dts/renesas/r8a77965.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77965.dtsi
@@ -1935,7 +1935,6 @@
 			max-frequency = <200000000>;
 			power-domains = <&sysc R8A77965_PD_ALWAYS_ON>;
 			resets = <&cpg 314>;
-			iommus = <&ipmmu_ds1 32>;
 			status = "disabled";
 		};
 
@@ -1948,7 +1947,6 @@
 			max-frequency = <200000000>;
 			power-domains = <&sysc R8A77965_PD_ALWAYS_ON>;
 			resets = <&cpg 313>;
-			iommus = <&ipmmu_ds1 33>;
 			status = "disabled";
 		};
 
@@ -1961,7 +1959,6 @@
 			max-frequency = <200000000>;
 			power-domains = <&sysc R8A77965_PD_ALWAYS_ON>;
 			resets = <&cpg 312>;
-			iommus = <&ipmmu_ds1 34>;
 			status = "disabled";
 		};
 
@@ -1974,7 +1971,6 @@
 			max-frequency = <200000000>;
 			power-domains = <&sysc R8A77965_PD_ALWAYS_ON>;
 			resets = <&cpg 311>;
-			iommus = <&ipmmu_ds1 35>;
 			status = "disabled";
 		};
 
-- 
2.17.1

